### PR TITLE
Conversion to material-ui

### PR DESCRIPTION
**Pull request by HendrikPN**
Almost complete conversion, except for some unused components: toast.js & CreateEnvironment.jsx & Repositories.jsx. I kept blueprint in the package.json since those files still exist.
Hero is still temporary.
Some logos and the banner are requested from the backend.

### CODE REVIEW

File: src/components/files/icons.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+// TODO: make svg icons and use SvgIcon as in https://v0.material-ui.com/#/components/svg-icon
+const GithubIcon = () => <img src='http://localhost:8000/static/icons/github-circle.png' height="48" width="48" alt=""/>;
+const SciGymIcon = () => <img src='http://localhost:8000/static/icons/scigym-logo.png' height="37" width="37" alt="" style={{padding: "5px"}}/>
+
+export {GithubIcon, SciGymIcon}
```
Comment (erik-sn) [inline src/components/files/icons.jsx]: you can do:

````
export const GithubIcon = () => ...
export const SciGymIcon = () => ...
```
and leave off the bottom export. It's a little easier to see inline that it's an exported variable.

File: src/components/files/images.jsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+
```
Comment (erik-sn) [inline src/components/files/images.jsx:2]: Can we consolidate these into one file, `icons.jsx` or `images.jsx`?

File: src/utils/toast.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { Position, Toaster } from "@blueprintjs/core";
 import { debounce } from "lodash";
 
+// not used anymore -- replace?
```
Comment (erik-sn) [inline src/utils/toast.js]: yep you can delete this.

**Comment by erik-sn** [comment]
Just a quick thing, instead of working on your master branch try to create feature branches. So `git checkout -b feature/conversion-to-material-ui`. Don't worry about this one but in the future lets move to something like that.

**Comment by HendrikPN** [comment]
Done! All images are now in /files/images.jsx in a proper format.
toast.js is gone

**Pull request merged**